django_project/base/management/commands/geocontext.json

```diff
@@ -2033,6 +2093,7 @@
             "key": "relative_humidity_group",
             "name": "Relative humidity Rasters",
             "description": "A collection of rasters depicting Daily minimum and maximum relative humidity  as relative humidity in South Africa",
+            "is_graph": false,
```

Comment:
I'm surprised to see this logic here. It is far better to separate the concerns of your PR's. Now we have logic for ArcRest services (one concern) and context groups (another concern) in the same PR. If you did the work for each in separate branches they could be reviewed separately (and your intention understood more easily) and merged independently. Also the VCS changeset would be clearer if we are rolling back through the history to understand why something broke or some behaviour changed.

Also I suggest to make the key here 'graphable' rather than 'is_graph' as it is more meaningful.

Reply:
I understand. I will be more careful regarding my pull requests and how I go about isolating my concerns.